### ISSUE TITLE
scylla start never ends while adding a new node to a cluster

### ISSUE THREAD

**Issue by fgelcer** [scylladb/scylla#7816]
*Installation details*
Scylla version (or git commit hash): 4.3.rc2-0.20201210.2148a194c21 with build-id e5eb54096ba980c90c06c4741b0356dea02dc4fa
Cluster size: 6
OS (RHEL/CentOS/Ubuntu/AWS AMI): ami-08aaddc100281dc7d

test is longevity 5000 tables.
in this test, 1 node is created, and then all tables are created for on it, and just then, the other 5 nodes are added to the cluster.

we have stress running, and started running nemesis (with the 5000 tables created, and all 6 nodes in the cluster)
```
/usr/bin/nodetool  status
Using /etc/scylla/scylla.yaml as the config file
Datacenter: eu-north
====================
Status=Up/Down
|/ State=Normal/Leaving/Joining/Moving
--  Address     Load       Tokens       Owns    Host ID                               Rack
UN  10.0.1.242  11.76 MB   256          ?       b9f261fe-983f-495b-9ed8-dd88ecaa0111  1a
UN  10.0.2.211  15.15 MB   256          ?       3d9f933a-9140-462a-af95-0209e83aeec2  1a
UN  10.0.1.39   14.45 MB   256          ?       20e17359-aa9d-4221-acb1-75b9e26ff88a  1a
UN  10.0.1.121  22.56 MB   256          ?       ce941c06-b4bd-49b4-a137-d700b0246c53  1a
UN  10.0.2.201  15.83 MB   256          ?       1a384060-2a34-45e3-a0e8-a0e5ac759904  1a
UN  10.0.3.110  ?          256          ?       91d0ab8c-1221-4327-be30-f6eca1ae5cf9  1a

Note: Non-system keyspaces don't have the same replication settings, effective ownership information is meaningless
```

then, we run grow cluster nemesis (it will add a node to existing cluster, and once it is done, it will decommission a node, to keep the cluster with the same size after the nemesis is done)
what happens is that the node is added, and starts bootstraping, but it never ends.

full logs can be found here:
```
+------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
|                                                                                                                  Log links for testrun with test id 8f3c83be-138b-4af3-a3c7-dc872c1c1f5b                                                                                                                   |
+-----------------+-------------+----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
| Date            | Log type    | Link                                                                                                                                                                                                                                                                       |
+-----------------+-------------+----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
| 20201220_091040 | grafana     | https://cloudius-jenkins-test.s3.amazonaws.com/8f3c83be-138b-4af3-a3c7-dc872c1c1f5b/20201220_091040/grafana-screenshot-overview-20201220_091040-longevity-5000-tables-5k-table-monitor-node-8f3c83be-1.png                                                                 |
| 20201220_091040 | grafana     | https://cloudius-jenkins-test.s3.amazonaws.com/8f3c83be-138b-4af3-a3c7-dc872c1c1f5b/20201220_091040/grafana-screenshot-scylla-jmx-crash-longevity-5000-tables-scylla-per-server-metrics-nemesis-20201220_091440-longevity-5000-tables-5k-table-monitor-node-8f3c83be-1.png |
| 20201220_093423 | db-cluster  | https://cloudius-jenkins-test.s3.amazonaws.com/8f3c83be-138b-4af3-a3c7-dc872c1c1f5b/20201220_093423/db-cluster-8f3c83be.zip                                                                                                                                                |
| 20201220_093423 | loader-set  | https://cloudius-jenkins-test.s3.amazonaws.com/8f3c83be-138b-4af3-a3c7-dc872c1c1f5b/20201220_093423/loader-set-8f3c83be.zip                                                                                                                                                |
| 20201220_093423 | monitor-set | https://cloudius-jenkins-test.s3.amazonaws.com/8f3c83be-138b-4af3-a3c7-dc872c1c1f5b/20201220_093423/monitor-set-8f3c83be.zip                                                                                                                                               |
| 20201220_093423 | sct-runner  | https://cloudius-jenkins-test.s3.amazonaws.com/8f3c83be-138b-4af3-a3c7-dc872c1c1f5b/20201220_093423/sct-runner-8f3c83be.zip                                                                                                                                                |
+-----------------+-------------+----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
```

important to mention that there were multiple `seastar_memory - oversized allocation` (probably related to #6376)

there are 373 stalls (few of them will be reported as separated issues)
```

**Comment by slivne** [comment]
@fgelcer did we run this on rc0 / rc1 ? did it pass - or is this the first run ?

**Comment by fgelcer** [comment]
@slivne we were blocked on rc0 with #7609, so this didn't happen as we failed before.

**Comment by slivne** [comment]
@fgelcer on 4.1 / 4.2 this test passed - just making sure - can you provide the info

**Comment by fgelcer** [comment]
> @fgelcer on 4.1 / 4.2 this test passed - just making sure - can you provide the info

checking the QA releases plan spreadsheet i can see that it never passed.
for 4.2 @aleksbykov had as a blocker #7439 but i don't see it now (it was postponed to 4.4).
for 4.1 @amoskong had few issues with c-s (similar to what i'm having in my latest job) but never fully passed AFAICT.

**Comment by avikivity** [comment]
Plenty of repair errors:

```
longevity-5000-tables-5k-table-db-node-8f3c83be-7/system.log:Dec 19 17:52:30 longevity-5000-tables-5k-table-db-node-8f3c83be-7 scylla[54255]:  [shard 14] repair - repair id [id=7, uuid=164731c7-0434-44a3-8904-021c4c82cb41] on shard 14 failed: std::runtime_error (repair id [id=7, uuid=164731c7-0434-44a3-8904-021c4c82cb41] on shard 14 failed to repair 261 sub ranges)
```

**Comment by fgelcer** [comment]
> Plenty of repair errors:
> 
> ```
> longevity-5000-tables-5k-table-db-node-8f3c83be-7/system.log:Dec 19 17:52:30 longevity-5000-tables-5k-table-db-node-8f3c83be-7 scylla[54255]:  [shard 14] repair - repair id [id=7, uuid=164731c7-0434-44a3-8904-021c4c82cb41] on shard 14 failed: std::runtime_error (repair id [id=7, uuid=164731c7-0434-44a3-8904-021c4c82cb41] on shard 14 failed to repair 261 sub ranges)
> ```

IIRC these repair failures are being seen in other tests as well

**Comment by roydahan** [comment]
@slivne this may be a regression.
@fgelcer we don't care about previous runs other non related issues that failed the run, please check if the same nemesis passed on previous runs or not.

**Comment by fgelcer** [comment]
> How long was it? maybe it's the same as you opened in [scylladb/scylla-cluster-tests#3027](https://github.com/scylladb/scylla-cluster-tests/issues/3027)?

it doesn't seem the same, as 24+ hours after there was no sign of the service being started, where on scylladb/scylla-cluster-tests#3027](https://github.com/scylladb/scylla-cluster-tests/issues/3027) it takes more than what SCT waits for it, but takes less than 1/2 hour

**Comment by roydahan** [comment]
Please check previous runs of 4.3 and 4.2 if this nemesis passed that stage of adding node and how long it took.

**Comment by denesb** [comment]
Closing as stale.
